src/compiler/tools/results/RefactorEditInfo.ts

```diff
@@ -45,5 +45,18 @@ export class RefactorEditInfo {
         return this.compilerObject.renameLocation;
     }
 
+    /**
+     * Executes the refactor edits.
+     *
+     * WARNING: This will cause all nodes to be forgotten in the changed files.
+     * @options - Options used when applying the changes.
+     */
+    applyChanges(options?: ApplyFileTextChangesOptions) {
```

Comment:
@dsherret I thought adding this one also here makes sense, right ?

Reply:
Yes it definitely does. Thanks! Want to add a separate PR for it though?

Reply:
Sure, on it